canonical/ubuntu.com
# fix: Allow pop up notifications to go under the navigation when scrolling

## Done

- Lower the z-index of notifications so they go under the navigation
- Put the notifications in their own file

## QA

- go to https://ubuntu-com-14222.demos.haus/#success and scroll down, see that the notification goes under the navigation

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-11728
